vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/WorkflowBusiness.java

```diff
@@ -394,7 +394,10 @@ public List<Simulation> getSimulationsWithGroupAdminRights(User user, String app
             workflows.addAll(getSimulationsAdminGroup(user.getEmail(), applications, wStatus, appClass, startDate, endDate, tag));
             workflows.addAll(workflowDAO.get(users, applications, wStatus, appClass, startDate, endDate, tag));
 
-            simulations = parseWorkflows(new ArrayList<>(workflows));
+            workflows = new ArrayList<>(workflows.stream().collect(
+                Collectors.toMap(Workflow::toString, w -> w, (e, r) -> e)).values());
```

Comment:
Maybe add a small comment to explain why you do that.
Also why not using the workflow-id as key ?